packages/perf/lib/index.js

```diff
@@ -47,12 +56,35 @@ class FirebasePerfModule extends FirebaseModule {
   constructor(...args) {
     super(...args);
     this._isPerformanceCollectionEnabled = this.native.isPerformanceCollectionEnabled;
+    this._instrumentationEnabled = true;
   }
 
   get isPerformanceCollectionEnabled() {
     return this._isPerformanceCollectionEnabled;
   }
 
+  get instrumentationEnabled() {
+    return this._instrumentationEnabled;
+  }
+
+  set instrumentationEnabled(enabled) {
+    if (!isBoolean(enabled)) {
+      throw new Error("firebase.perf().instrumentationEnabled = 'enabled' must be a boolean.");
+    }
+    this._instrumentationEnabled = enabled;
```

Comment:
don't look now, but iOS actually has a native API for this. it may not be implemented in native yet but is likely trivial (should mirror the setPerformanceCollectionEnabled API...) but should be implemented and should be called in this setter